### DISCUSSION TITLE
Stomp cannot start

### DISCUSSION THREAD

**Discussion by z331235161** [rabbitmq/rabbitmq-server#3193] [Other] [answered]
[root@localhost ~]# systemctl status rabbitmq-server.service
● rabbitmq-server.service - RabbitMQ broker
   Loaded: loaded (/usr/lib/systemd/system/rabbitmq-server.service; disabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since 四 2021-07-15 11:15:11 CST; 7min ago
  Process: 8331 ExecStart=/usr/lib/rabbitmq/bin/rabbitmq-server (code=exited, status=1/FAILURE)
 Main PID: 8331 (code=exited, status=1/FAILURE)

7月 15 11:15:08 localhost.localdomain rabbitmq-server[8331]: {rabbit_stomp,start,[normal,[]]}}}
7月 15 11:15:08 localhost.localdomain rabbitmq-server[8331]: Log files (may contain more information):
7月 15 11:15:08 localhost.localdomain rabbitmq-server[8331]: /var/log/rabbitmq/rabbit@localhost.log
7月 15 11:15:08 localhost.localdomain rabbitmq-server[8331]: /var/log/rabbitmq/rabbit@localhost-sasl.log
7月 15 11:15:09 localhost.localdomain rabbitmq-server[8331]: {"init terminating in do_boot",{rabbit,failure_during_boot,{could_not_start,rabbitmq_stomp,{{shutdown,{failed_to_start_child,'rabbit_stomp_listener_sup_:::61613'...normal,[]]}}}}}
7月 15 11:15:11 localhost.localdomain rabbitmq-server[8331]: Crash dump was written to: erl_crash.dump
7月 15 11:15:11 localhost.localdomain systemd[1]: rabbitmq-server.service: main process exited, code=exited, status=1/FAILURE
7月 15 11:15:11 localhost.localdomain systemd[1]: Failed to start RabbitMQ broker.
7月 15 11:15:11 localhost.localdomain systemd[1]: Unit rabbitmq-server.service entered failed state.
7月 15 11:15:11 localhost.localdomain systemd[1]: rabbitmq-server.service failed.
Hint: Some lines were ellipsized, use -l to show in full.
[root@localhost ~]# lsof -i tcp:61613
[root@localhost ~]# netstat   -nultp
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name    
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      951/sshd            
tcp        0      0 127.0.0.1:25            0.0.0.0:*               LISTEN      1248/master         
tcp6       0      0 :::22                   :::*                    LISTEN      951/sshd            
tcp6       0      0 ::1:25                  :::*                    LISTEN      1248/master         
udp        0      0 127.0.0.1:323           0.0.0.0:*                           700/chronyd         
udp6       0      0 ::1:323                 :::*                                700/chronyd

**Comment by michaelklishin** [comment, marked as answer]
The message suggests checking the log files (`/var/log/rabbitmq/rabbit@localhost.log`, `/var/log/rabbitmq/rabbit@localhost-sasl.log` — this one indicates you are running a 3.6 version?). There isn't enough information for us to recommend anything specific.

**Reply by z331235161** [reply]
Thank you. Switch to 3.8 verison and start successfully
